sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -2294,19 +2737,26 @@ def Read(self, path, typ, id, initial_headers, options=None, **kwargs):  # pylin
             options = {}
 
         initial_headers = initial_headers or self.default_headers
-        headers = base.GetHeaders(self, initial_headers, "get", path, id, typ, options)
+        headers = base.GetHeaders(self, initial_headers, "get", path, resource_id, resource_type, options)
         # Read will use ReadEndpoint since it uses GET operation
-        request_params = _request_object.RequestObject(typ, documents._OperationType.Read)
+        request_params = RequestObject(resource_type, documents._OperationType.Read)
         result, self.last_response_headers = self.__Get(path, request_params, headers, **kwargs)
         return result
 
-    def DeleteResource(self, path, typ, id, initial_headers, options=None,
-                       **kwargs):  # pylint: disable=redefined-builtin
+    def DeleteResource(
+        self,
+        path: str,
+        resource_type: str,
+        resource_id: Optional[str],
```

Comment:
These parameter renames could potentially be considered breaking if we consider that this object was mistakenly made public....

Reply:
I've reverted the name changes just to reduce the changes.